.github/actions/secure-checkout/action.yaml

```diff
@@ -6,12 +6,10 @@ inputs:
     description: "The repository to check out."
     required: false
     default: ${{ github.repository }}
-  sha:
-    description: "The SHA to checkout."
+  ref:
+    description: "The ref or SHA to checkout."
```

Comment:
nit: document that ref should be fully-formed in the form `refs/*, and that it can handle branches and tags.
currently it also handles pull requests. I'm not sure why that would be used in practice

Reply:
Documenting for later: we'll keep in branches, technically I think we only need it for tags and otherwise we're using sha